contracts/scripts/files/mainnetConfig.json

```diff
@@ -0,0 +1,46 @@
+{
+  "contracts": {
+    "l1Timelock": "0xE6841D92B0C345144506576eC13ECf5103aC7f49",
+    "rollup": "0x5eF0D09d1E6204141B4d37530808eD19f60FBa35",
+    "bridge": "0x8315177aB297bA92A06054cE80a67Ed4DBd7ed3a",
+    "sequencerInbox": "0x1c479675ad559DC151F6Ec7ed3FbF8ceE79582B6",
+    "rollupEventInbox": "0x57Bd336d579A51938619271a7Cc137a46D0501B1",
+    "outbox": "0x0B9857ae2D4A3DBe74ffE1d7DF045bb7F96E4840",
+    "inbox": "0x4Dbd4fc535Ac27206064B68FfCf827b0A60BAB3f",
+    "osp": "0x3E1f62AA8076000c3218493FE3e0Ae40bcB9A1DF"
```

Comment:
beyond this PR but we actually need to upgrade the osp since there are [some changes](https://github.com/OffchainLabs/nitro-contracts/pull/56/files#diff-4fe09f06440b75b49e812bd2e303043b70e765a0968fe089b7b60a0eacc1289d)

Reply:
redeploy osp as part of setup